pkg/azurefile/controllerserver.go

```diff
@@ -704,3 +724,28 @@ func (d *Driver) snapshotExists(ctx context.Context, sourceVolumeID, snapshotNam
 
 	return false, azfile.ShareItem{}, nil
 }
+
+// isValidVolumeCapabilities validates the given VolumeCapability array is valid
+func isValidVolumeCapabilities(volCaps []*csi.VolumeCapability) error {
+	if len(volCaps) == 0 {
+		return fmt.Errorf("CreateVolume Volume capabilities must be provided")
+	}
+	hasSupport := func(cap *csi.VolumeCapability) error {
+		if blk := cap.GetBlock(); blk != nil {
```

Comment:
does this mean Azure File driver does not support block volume, right?

Reply:
I think Azure File should not support block volume, block volume only works for disk

Reply:
To my understanding it should not support raw block volume. But correct me if I am wrong.

Reply:
code part LGTM, thanks.
so the PR title could be `fix: return error for creating raw block volume(not supported)` ?

Reply:
Done. Thanks